README.md

```diff
@@ -25,9 +25,15 @@ rails server
 
 And browse to `localhost:3000`
 
+### Tests
+
+Dependencies:
+
+- Geckodriver
+- Firefox
```

Comment:
The feature tests are running in Chrome for me, so I assume they're using Chromedriver instead? It might pay to put the version number, too.

Reply:
Probably should be Chrome rather than Firefox?

Reply:
Fixed in commit 603acd23e22678639e18d09dace2bb437aafe4f7